lib/matplotlib/__init__.py

```diff
@@ -1140,12 +1140,15 @@ def use(backend, *, force=True):
         # Nothing to do if the requested backend is already set
         pass
     else:
-        try:
-            from matplotlib import pyplot as plt
-            plt.switch_backend(name)
-        except ImportError:
-            if force:
-                raise
+        if 'matplotlib.pyplot' in sys.modules:
```

Comment:
There should be a comment why we do this dance; essentially some version of the PR description.